src/github/client/github-client.ts

```diff
@@ -1,38 +1,21 @@
 import Logger from "bunyan";
-import { Octokit } from "@octokit/rest";
-import axios, { AxiosInstance, AxiosRequestConfig, AxiosResponse } from "axios";
+import {Octokit} from "@octokit/rest";
+import axios, {AxiosInstance, AxiosRequestConfig, AxiosResponse} from "axios";
 import AppTokenHolder from "./app-token-holder";
 import InstallationTokenCache from "./installation-token-cache";
 import AuthToken from "./auth-token";
-import { GetPullRequestParams } from "./types";
-import { handleFailedRequest, instrumentFailedRequest, instrumentRequest, setRequestStartTime } from "./interceptors";
-import { metricHttpRequest } from "../../config/metric-names";
-import { getLogger } from "../../config/logger";
-import { urlParamsMiddleware } from "../../util/axios/common-middleware";
-import { InstallationId } from "./installation-id";
+import {GetPullRequestParams} from "./types";
+import {handleFailedRequest, instrumentFailedRequest, instrumentRequest, setRequestStartTime} from "./interceptors";
+import {metricHttpRequest} from "../../config/metric-names";
+import {getLogger} from "../../config/logger";
+import {urlParamsMiddleware} from "../../util/axios/common-middleware";
+import {InstallationId} from "./installation-id";
 import {GetBranchesQuery, GetBranchesResponse, ViewerRepositoryCountQuery} from "./github-queries";
+import {GraphQLError, GraphQLErrors, rateLimitErrorFromResponse} from "./errors";
 
-/**
- * The response type for GitHub GraphQL calls.
- * Was copied from @octokit/graphql to avoid adding a dependency on Octokit Graphql client
- */
```

Comment:
This comment doesn't make sense anymore.

I've extracted GraphQLErrors as a separate type and moved it to errors.ts, also added "type" parameter, because it was missing.